### PR TITLE
Enhance Rendertron config

### PR DESCRIPTION
**Pull request by gravi2**
This PR enhances the config options exposed/available to rendertron.  It adds the following config options in addition to the existing 'datastoreCache' option

 ```
timeout: number;
port: string;
width: number;
height: number;
```
Respective default values are also been added:
```
datastoreCache: false,
timeout: 10000,
port: '3000',
width: 1000,
height: 1000,
```
This PR helps address following open issues:
1. #213 , #184 (Allow setting viewport/resolution) 
2. #160, #211  (Allow setting timeout)
3. #253 ( Specify port in config ) 

Also includes following Improvements:
1. npm run monitor and monitor-inspect will now listen for any src/*.ts file changes, build the rendertron.js file and reload the server. This helps for smoother development process.   
2. Added nodemon.json config file to control nodemon options. 
3. Added documentation in README for the new config options.

### CODE REVIEW

**Comment by gravi2** [comment]
@AVGP: do you have any feedback on the PR or anything I can help to get the PR merged?

File: package.json [hunk 1]
```diff
@@ -47,15 +47,16 @@
...
     "@types/koa-route": "^3.2.4",
     "@types/koa-send": "^4.1.1",
     "@types/koa-static": "^4.0.0",
     "@types/puppeteer": "^1.6.0",
     "@types/supertest": "^2.0.4",
-    "@types/koa-logger": "^3.1.1",
     "ava": "1.0.0-beta.6",
     "nodemon": "^1.18.3",
     "supertest": "^3.0.0",
+    "ts-node": "^8.0.2",
```
Comment (AVGP) [inline package.json]: Why do we need this additional dependency?

Comment (gravi2) [reply package.json]: @AVGP I have removed that dependency as it is no longer needed. Also I have reverted the package-lock.json file to the one from master. See my comment below for details. I will let you mark this review comment resolved.

**Comment by gravi2** [comment]
@AVGP I have removed the ts-node dependency as requested. I might have added it for something but we longer seem to need it.
Also BTW I have reverted the package-lock.json file to that of master, since I see there is existing issue in master. See my comment in the issue linked below:

https://github.com/GoogleChrome/rendertron/issues/281#issuecomment-472319405

**Comment by AVGP** [comment]
Awesome thank you, @gravi2

**Pull request merged**